FLEDGE.md

```diff
@@ -148,7 +148,10 @@ const myAuctionConfig = {
   'auctionSignals': {...},
   'sellerSignals': {...},
   'sellerTimeout': 100,
-  'perBuyerSignals': {'https://www.example-dsp.com': {...},
+  'sellerExperimentGroupId': 12345,
+  'perBuyerSignals': {'https://www.example-dsp.com': {
```

Comment:
So what CL I have pending implements has it as a separate perBuyerExperimentGroupId map, similar to perBuyerTimeouts, and also admitting * for setting the default. What do you think of that interface?

Reply:
My thought is that if it is a separate field then it's one more signal a seller has to map from the contextual call to `runAdAuction()`. This may be okay as a one-off, but if you can imagine more buyer-controlled fields being added in the future that the browser needs to understand then it's a change all the sellers need to handle as well. Also need to worry about field name conflicts if they're root level in perBuyerSignals.

Maybe a separate `perBuyerConfigurations` object that for now just contains the `experimentGroupId`, but could, in the future, be extended to take any additional buyer-controlled fields that the browser needs to know about?

Reply:
https://github.com/WICG/turtledove/pull/276 also runs into this, with `perBuyerGroupLimits`

On the other hand, with the way partial updating of auction configurations works, there is some advantage of keeping them separate to allow more granular updates.

Reply:
Is there anything required on my end?

Reply:
@jeffkaufman could you please expand on "the way partial updating of auction configurations works"?

I like the idea of a `perBuyerConfigurations` object that contains all buyer-controlled FLEDGE settings like the experimentGroupId (and buyer limits/timeouts, maybe an exploration factor for IG selection, opt-in for bandit algorithm, I'm just making stuff up here). 

The thought is that if an SSP does
```
foreach dsp:
    perBuyerConfigurations[dsp] = contextualResponse[dsp].buyerConfiguration
```
then buyers don't have to wait for SSP changes to utilize new buyer-controlled FLEDGE parameters if/when FLEDGE adds them.

Reply:
> could you please expand on "the way partial updating of auction configurations works"?

Sorry, I got confused and thought we were talking about interest groups.  Ignore me!

Reply:
Which means I'm also in favor of a `perBuyer` section.  Might as well merge this with `interestGroupBuyers` while we're at it.

Reply:
Note that I've landed https://chromiumdash.appspot.com/commit/9c37c816537202ef21e9447ddfe081093d292c97 which supports this via a separate field, but the intent was very much to incorporate this feedback on better ways of phrasing this in API once a consensus coalesces...